report_thesis/src/sections/background/preprocessing/min-max.tex

```diff
@@ -1,10 +1,12 @@
 \subsubsection{Min-Max Normalization}\label{subsec:min-max}
-Min-max normalization rescales the range of features to $[0, 1]$ or $[a, b]$, where $a$ and $b$ represent the new minimum and maximum values, respectively.
+Min-max normalization rescales the range of features to a specific range $[a, b]$, where $a$ and $b$ represent the new minimum and maximum values, respectively.
 The goal is to normalize the range of the data to a specific scale, typically 0 to 1.
-Mathematically, min-max normalization is defined as:
+The min-max normalization of a feature vector $\mathbf{x}$ is given by:
+
 $$
-	v' = \frac{v - \min(F)}{\max(F) - \min(F)} \times (b - a) + a,
+x'_i = \frac{x_i - \min(\mathbf{x})}{\max(\mathbf{x}) - \min(\mathbf{x})}(b - a) + a,
 $$
-where $v$ is the original value, $\min(F)$ and $\max(F)$ are the minimum and maximum values of the feature $F$, respectively.
 
-This type of normalization is beneficial because it ensures that each feature contributes equally to the analysis, regardless of its original scale.
\ No newline at end of file
+where $x_i$ is the original value, $\min(\mathbf{x})$ and $\max(\mathbf{x})$ are the minimum and maximum values of the feature vector $\mathbf{x}$, respectively, and $x'_i$ is the normalized feature value.
```

Comment:
Is it redundant to repeat that its a feature vector x? In the introduction you said that this is the case.

Reply:
I have done this for every section on preprocessing to make absolutely sure that there is no ambiguity